extensions/blocks/gif/index.js

```diff
@@ -58,4 +58,12 @@ export const settings = {
 	},
 	edit,
 	save: () => null,
+	example: {
+		attributes: {
+			align: 'center',
+			giphyUrl: 'https://giphy.com/embed/fxKWgoOG9hzPPkE1oc',
+			paddingTop: '100%',
+			searchText: __( 'WordPress', 'jetpack' ),
```

Comment:
Does this need to be translated?

Reply:
I suppose not since it doesn't visibly show up. Will remove.